src/app/domain/case-viewer/case-viewer.module.ts

```diff
@@ -7,9 +7,11 @@ import { SummaryPanelComponent } from './components/summary-panel/summary-panel.
 import { PartiesPanelComponent } from './components/parties-panel/parties-panel.component';
 import { CaseNavComponent } from './components/case-nav/case-nav.component';
 import { TimelinePanelComponent } from './components/timeline-panel/timeline-panel.component';
+import { TimelineComponent } from './components/timeline/timeline.component';
 import { CaseDetailsBarComponent } from './components/case-details-bar/case-details-bar.component';
 import {DocumentViewerModule} from '../../shared/components/document-viewer/document-viewer.module';
 import {RouterModule} from '@angular/router';
+import { TimelineSectionComponent } from './components/timeline-section/timeline-section.component';
```

Comment:
This doesn't need to be seperate component. Simply put the surrounding HTML in the parent container and use the re-usable timeline component

Reply:
removed